.github/workflows/build.yml

```diff
@@ -118,7 +118,7 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17 ]
-        junit-version: [ '5.8.2' ]
+        junit-version: [ '5.9.0' ]
```

Comment:
Some of these changes are unrelated to the actual issue - do you think we could separate these from the PR?

Reply:
The change came from the merge, Git history clearly shows that to me 🤷‍♂️

Reply:
You have to merge in `main`, then they will disappear.